src/main/webapp/web.xml

```diff
@@ -7,8 +7,110 @@
+    </servlet>
+
+    <servlet>
+        <servlet-name>DriverAddingToCarController</servlet-name>
+        <servlet-class>mate.controller.car.AddDriverToCarController</servlet-class>
+    </servlet>
+
+    <servlet>
+        <servlet-name>CarDefaultCreatingController</servlet-name>
+        <servlet-class>mate.controller.car.CreateCarDefaultController</servlet-class>
+    </servlet>
+
+    <servlet>
+        <servlet-name>DisplayCarsController</servlet-name>
+        <servlet-class>mate.controller.car.DisplayCarsController</servlet-class>
+    </servlet>
+
+    <servlet>
+        <servlet-name>DeleteCarsController</servlet-name>
+        <servlet-class>mate.controller.car.DeleteCarsController</servlet-class>
+    </servlet>
+
+    <servlet>
+        <servlet-name>ManufacturerCreatingController</servlet-name>
+        <servlet-class>mate.controller.manufacturer.CreateManufacturerController</servlet-class>
+    </servlet>
+
+    <servlet>
+        <servlet-name>DisplayManufacturersController</servlet-name>
+        <servlet-class>mate.controller.manufacturer.DisplayManufacturersController</servlet-class>
+    </servlet>
+
+    <servlet>
+        <servlet-name>DeleteManufacturerController</servlet-name>
+        <servlet-class>mate.controller.manufacturer.DeleteManufacturerController</servlet-class>
+    </servlet>
+
     <servlet-mapping>
         <servlet-name>index</servlet-name>
         <url-pattern>/index</url-pattern>
     </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>DriverCreatingController</servlet-name>
+        <url-pattern>/driver/create</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>DeleteDriverController</servlet-name>
+        <url-pattern>/driver/delete</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>DriverDisplayController</servlet-name>
+        <url-pattern>/driver/display</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>DriverAddingToCarController</servlet-name>
+        <url-pattern>/car/driver/add</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>ManufacturerCreatingController</servlet-name>
+        <url-pattern>/manufacturer/create</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>CarDefaultCreatingController</servlet-name>
+        <url-pattern>/car/create</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>DisplayCarsController</servlet-name>
+        <url-pattern>/car/display</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>DeleteCarsController</servlet-name>
+        <url-pattern>/car/delete</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>DisplayManufacturersController</servlet-name>
+        <url-pattern>/manufacturer/display</url-pattern>
```

Comment:
no need to add dispplay here just manufacturers is enough Same everywhere

Reply:
I understand, but I ll wait for mentor response for this